backend/app/notify/email.go

```diff
@@ -295,8 +212,8 @@ func (e *Email) buildVerificationMessage(user, email, token, site string) (strin
 	return e.buildMessage(subject, msg.String(), email, "text/html", "")
 }
 
-// buildMessageFromRequest generates email message based on Request using e.MsgTemplate
-func (e *Email) buildMessageFromRequest(req Request, forAdmin bool) (string, error) {
+// buildReplyRequest generates email message based on Request using e.replyTmpl
```

Comment:
It's building a message from the request, not reply request. I would suggest undoing renames if you have no strong feelings about them, as new ones don't feel better than old ones.

Reply:
My logic in this:
- we have two types of emails in our service
- first for verification and second for what?
- this template notifies users/admins about replies

1. it is a reply to a user comment
2. it is a reply to an admin article on site

P.s. I made wrong naming I wanted to name it like `buildReplyMessage`
I haven't understood why this email builds by request.

Reply:
`buildReplyMessage` would be appropriate name